Development.md

```diff
@@ -142,6 +145,31 @@ clang -Xclang -ast-dump=json file.c
 
 Another option is to use clang-query ([tutorial](https://devblogs.microsoft.com/cppblog/exploring-clang-tooling-part-2-examining-the-clang-ast-with-clang-query/)).
 
+### Automated test case reduction
+
+In case of a crash, it may be possible to automatically reduce
+it using [C-Reduce](https://github.com/csmith-project/creduce).
+
+**Important:**
+On macOS, use `brew install --HEAD creduce`,
+as the default version is very outdated.
+
+There is a helper script [tools/reduce.py](/tools/reduce.py)
+which can coordinate `scip-clang` and `creduce`, since getting
+the details right is a bit finicky in the general case.
```

Comment:
"The details" is mysterious. Omit for brevity, or elaborate for posterity.

Reply:
Added more details.